packages/flutter/lib/src/services/binding.dart

```diff
@@ -105,14 +100,14 @@ mixin ServicesBinding on BindingBase, SchedulerBinding {
     //      https://github.com/dart-lang/sdk/issues/31960
     // TODO(ianh): Remove this complexity once these bugs are fixed.
     final Completer<String> rawLicenses = Completer<String>();
-    Timer.run(() async {
+    scheduleTask(() async {
       rawLicenses.complete(rootBundle.loadString('LICENSE', cache: false));
-    });
+    }, Priority.animation);
```

Comment:
Not sure whether this is the right priority

Reply:
Seems like the right priority.